.travis.yml

```diff
@@ -15,11 +18,20 @@ env:
     - SCRIPT=test-maven
     - SCRIPT=test-documentation
     - SCRIPT=test-code-style
+
 # important to use eval, otherwise "&&" is passed as an argument to sbt rather than being processed by bash
 script: bin/$SCRIPT
 before_install:
     # See https://github.com/travis-ci/travis-ci/issues/4629#issuecomment-239493916
     - rm ~/.m2/settings.xml
+    - if [[ "$TRAVIS_JDK_VERSION" == "oraclejdk9" ]]; then
```

Comment:
Try adding `set -eo pipefail` to this to ensure that it fails the build on an error. By default, bash just happy continues along running commands in a shell script, and uses the exit status of the final command as the exit status for the entire process. In this case, the final command (`export JAVA_HOME=/usr/lib/jvm/java-9-oracle`) will always succeed.